std/assembly/typedarray.ts

```diff
@@ -34,6 +80,29 @@ export class Uint8ClampedArray extends TypedArray<u8,u32> {
   subarray(begin: i32 = 0, end: i32 = 0x7fffffff): Uint8ClampedArray {
     return changetype<Uint8ClampedArray>(super.subarray(begin, end));
   }
+
+  /**
+   * @param callbackfn {function} - a function that reduces each value to a ReturnType
+   * @param initialvalue {ReturnType} - the initial ReturnType value to be passed to the callbackfn
+   */
+  reduce<ReturnType>(
+    callbackfn: (accumulator: ReturnType, value: u8, index: i32, array: Uint8ClampedArray) => ReturnType,
+    initialvalue: ReturnType,
```

Comment:
plz change to `initialValue` for consistency

Reply:
Actually I copied some stuff from mdn directly and it was a formatting issue. Thanks for the catch.